releases/release-5.3.2.md

```diff
@@ -0,0 +1,243 @@
+
+## __unsorted
+
++ PingCAP/TiDB
+
+    - release-note [#33892](https://github.com/pingcap/tidb/issues/33892)
+    - release-note [#34906](https://github.com/pingcap/tidb/issues/34906)
+    - release-note [#35340](https://github.com/pingcap/tidb/issues/35340)
+    - release-note [#29283](https://github.com/pingcap/tidb/issues/29283)
+    - release-note [#34320](https://github.com/pingcap/tidb/issues/34320)
+    - release-note [#34417](https://github.com/pingcap/tidb/issues/34417)
+    - release-note [#34350](https://github.com/pingcap/tidb/issues/34350)
+    (dup: release-6.1.0.md > Bug fixes> Tools> TiDB Lightning)- Fix the issue of Local-backend import failure caused by out-of-bounds data in the `auto_increment` column [#27937](https://github.com/pingcap/tidb/issues/27937)
+    (dup: release-6.1.0.md > Improvements> Tools> TiDB Lightning)- Optimize Scatter Region to batch mode to improve the stability of the Scatter Region process [#33618](https://github.com/pingcap/tidb/issues/33618)
+    (dup: release-5.4.1.md > Bug Fixes> Tools> TiDB Lightning)- Fix the issue that the precheck does not check local disk resources and cluster availability [#34213](https://github.com/pingcap/tidb/issues/34213)
+    (dup: release-5.4.1.md > Bug Fixes> Tools> Backup & Restore (BR))- Fix duplicate primary keys when inserting a record into a table after incremental restoration [#33596](https://github.com/pingcap/tidb/issues/33596)
+    - release-note [#33335](https://github.com/pingcap/tidb/issues/33335)
+    (dup: release-5.4.1.md > Bug Fixes> Tools> Backup & Restore (BR))- Fix the issue that schedulers do not resume after BR or TiDB Lightning exits abnormally [#33546](https://github.com/pingcap/tidb/issues/33546)
+    - Fix the issue that NewCollationEnable config not checked during restoration. [#33422](https://github.com/pingcap/tidb/issues/33422)
+    (dup: release-5.4.1.md > Bug Fixes> Tools> Backup & Restore (BR))- Fix a bug that BR incremental restore returns errors mistakenly due to DDL jobs with empty query [#33322](https://github.com/pingcap/tidb/issues/33322)
+    (dup: release-5.4.1.md > Bug Fixes> Tools> Backup & Restore (BR))- Fix the issue that BR does not retry enough times when Regions are not consistent during restoration [#33419](https://github.com/pingcap/tidb/issues/33419)
+    - release-note [#33310](https://github.com/pingcap/tidb/issues/33310)
+    (dup: release-6.0.0-dmr.md > Bug fixes> Tools> Backup & Restore (BR))- Fix a bug that BR gets stuck when the restore operation meets some unrecoverable errors [#33200](https://github.com/pingcap/tidb/issues/33200)
+    (dup: release-5.4.1.md > Bug Fixes> Tools> TiDB Lightning)- Fix the checksum error "GC life time is shorter than transaction duration" [#32733](https://github.com/pingcap/tidb/issues/32733)
+    (dup: release-5.4.1.md > Bug Fixes> Tools> Backup & Restore (BR))- Fix the issue that BR fails to back up RawKV [#32607](https://github.com/pingcap/tidb/issues/32607)
+
++ TiKV/TiKV
```

Comment:
TiKV

## Bug Fixes

- Fix the bug that causes frequent pd client reconnection [#12345](https://github.com/tikv/tikv/issues/12345)
- Fix a wrong check in datetime when the datetime has a fraction and 'Z' [#12739](https://github.com/tikv/tikv/issues/12739)
- Fix the bug that converting empty string causes TiKV crash [#12673](https://github.com/tikv/tikv/issues/12673)
- Fix possible duplicate commit records in async-commit pessimistic transactions [#12615](https://github.com/tikv/tikv/issues/12615)
- Fix the bug that TiKV reports the `invalid store ID 0` error when using Follower Read [#12478](https://github.com/tikv/tikv/issues/12478)
- Fix the issue of TiKV panic caused by the race between destroying peers and batch splitting Regions [#12368](https://github.com/tikv/tikv/issues/12368)
- Fix the issue that successfully committed optimistic transactions may report the `Write Conflict` error when the network is poor [#34066](https://github.com/pingcap/tidb/issues/34066)
- Fix the issue that TiKV panics and destroys peers unexpectedly when the target Region to be merged is invalid [#12232](https://github.com/tikv/tikv/issues/12232)
- Fix the bug that stale messages cause TiKV panic [#12023](https://github.com/tikv/tikv/issues/12023)
- Fix the issue of intermittent packet loss and out of memory (OOM) caused by the overflow of memory metrics [#12160](https://github.com/tikv/tikv/issues/12160)
- Fix a potential panic issue when TiKV performs profiling on Ubuntu 18.04 [#9765](https://github.com/tikv/tikv/issues/9765)
- Fix the issue that tikv-ctl returns an incorrect result due to incorrect string match pattern [#12329](https://github.com/tikv/tikv/issues/12329)
- Fix a bug that replica reads might violate the linearizability [#12109](https://github.com/tikv/tikv/issues/12109)
- Fix the TiKV panic issue that the target peer is replaced with the peer that is destroyed without being initialized when merging a Region [#12048](https://github.com/tikv/tikv/issues/12048)
- Fix a bug that TiKV might panic if it has been running for 2 years or more [#11940](https://github.com/tikv/tikv/issues/11940) 

## Improvement

- Improve the health check to detect unavailable Raftstore, so that the TiKV client can update Region Cache in time [#12398](https://github.com/tikv/tikv/issues/12398)
- Transfer the leadership to CDC observer to reduce latency jitter [#12111](https://github.com/tikv/tikv/issues/12111)
- Reduce the write latency by separating I/O operations from Raftstore thread pool (disabled by default). For more information about tuning, see [Tune TiKV Thread Pool Performance](/tune-tikv-thread-performance.md) [#10540](https://github.com/tikv/tikv/issues/10540)
- Add more metrics for the garbage collection module of Raft logs [#11374](https://github.com/tikv/tikv/issues/11374)

Reply:
Added in [4bace2a](https://github.com/pingcap/docs/pull/9029/commits/4bace2a484b32e4c1cdd16c995ce0708f96bd96b)